edx_rbac/utils.py

```diff
@@ -50,7 +52,7 @@ def get_decoded_jwt_from_request(request):
     Returns a decoded jwt dict if it finds it.
     Returns a None if it does not.
     """
-    jwt_cookie = request.COOKIES.get(jwt_cookie_name(), None)
+    jwt_cookie = request.COOKIES.get(jwt_cookie_name(), None) or request.auth
```

Comment:
Had a look here. I suspect for the sake of testing it might make sense just to mock request.auth (because tests dont use the same auth as in prod)